front/lib/api/extract.ts

```diff
@@ -154,3 +154,37 @@ export async function getExtractedEvents(
     };
   });
 }
+
+export async function deleteExtractedEvent(
+  auth: Authenticator,
+  id: string
+): Promise<boolean> {
+  const owner = auth.workspace();
+  if (!owner) {
+    throw "extracted_event_auth_error";
+  }
+
+  const event = await ExtractedEvent.findOne({
+    where: {
+      id: id,
+    },
+    include: [
+      {
+        model: EventSchema,
+        as: "eventSchema",
+      },
+    ],
+  });
+
+  if (!event) {
+    throw "extracted_event_not_found";
```

Comment:
This should not be a throw error I think. We can just return here no?

Reply:
Basically I did that to be able to return different `apiError` types depending on why we couldn't delete, but I get that we don't want to throw as is, I'll add a commit!

Reply:
Fine to use a Result here as well :+1: